doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
+
+Notifications include two fields: `subscriptionId` (which value is the one of the subscription triggering
+the notification) and `data` (which holds the actual data corresponding to entities and attributes).
+
+The `data` field can use different representation depending on two properties (especified as part of the
+corresponding subscription, see subscriptions operations section for details).
+
+* attrsFormat ("full", "keyValues", "values"; default is "full") to specify how the attributes are
+  represented. See " Simplified Entity Representation" for details.
+* dataFormat ("array", "object"; default is "array"): to specify if the `data` field holding the information
+  is an array or an object.
+  * In "array" case, "data" is an array which elements are objects, each one representing an entity.
+  * In "object" case, "data" in an object which keys are entity types and which values are object which keys, in sequence,
+    are entity ids and which values is an object representing the entity.
+
+Combining both, there are 6 possibilities:
```

Comment:
From a notification receiver point of view, it is much easier to know the format being used looking to an HTTP header that looking to the payload and start guessing based in its content.